lib/progress_bar/progress_bar_small/progress_bar_small_widget.dart

```diff
@@ -0,0 +1,179 @@
+        //
+        // `isSecondaryText` (bool): Defaults to false. Controls visibility of `secondaryText`.
+        //
+        // `isTertiaryText` (bool): Defaults to false. Controls visibility of `tertiaryText`.
+        //
+        // `isIcon` (bool): Defaults to false. Controls visibility of the icon.
+        Column(
+      mainAxisSize: MainAxisSize.min,
+      children: [
+        Align(
+          alignment: const AlignmentDirectional(-1.0, 0.0),
+          child: Row(
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.end,
+            children: [
+              Align(
+                alignment: const AlignmentDirectional(0.0, 0.0),
+                child: Row(
+                  mainAxisSize: MainAxisSize.max,
+                  crossAxisAlignment: CrossAxisAlignment.end,
+                  children: [
+                    if (widget.isPrimaryText)
+                      Text(
+                        valueOrDefault<String>(
+                          widget.primaryText,
+                          '75',
+                        ),
+                        style: FlutterFlowTheme.of(context).bodyMedium.override(
+                              fontFamily: 'Nunito',
+                              color: widget.primaryTextColor,
+                            ),
+                      ),
+                    if (widget.isSecondaryText)
+                      Text(
+                        valueOrDefault<String>(
+                          widget.secondaryText,
+                          '/ 50 L',
+                        ),
+                        style: FlutterFlowTheme.of(context).labelSmall.override(
+                              fontFamily: 'Nunito',
+                              color: widget.secondaryTextColor,
+                            ),
+                      ),
+                  ].divide(const SizedBox(width: 4.0)),
+                ),
+              ),
+              Flexible(
+                child: Align(
+                  alignment: const AlignmentDirectional(1.0, 0.0),
+                  child: Row(
+                    mainAxisSize: MainAxisSize.min,
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    crossAxisAlignment: CrossAxisAlignment.end,
+                    children: [
+                      if (widget.isTertiaryText)
+                        Text(
+                          valueOrDefault<String>(
+                            widget.tertiaryText,
+                            'Extend',
+                          ),
+                          style:
+                              FlutterFlowTheme.of(context).labelSmall.override(
+                                    fontFamily: 'Nunito',
+                                    color: widget.tertiaryTextColor,
+                                  ),
+                        ),
+                      if (widget.isIcon) widget.icon!,
```

Comment:
here icon can be null which can cause the code to break.
```suggestion
                    if(widget.icon) widget.icon,
```